course-details.md

```diff
@@ -1,9 +1,43 @@
-This course is about continuous delivery, or CD, with GitHub Actions. We create two workflows to deploy our app to an AWS S3 bucket automatically. CD is an important part of modern software development, and it has a big meaning. CD is the practice of delivering software faster and with higher quality. To do this, it takes many different practices, behaviors, and technologies. 
+This course is about continuous delivery, or CD, with GitHub Actions. CD is a crucial part of modern software development, and it can have a monumental impact on development projects. 
 
-In this course, we focus on workflows to deploy pull requests automatically to a staging environment, and to deploy all merged commits to production. In this course, you will:
+Continuous Delivery is the practice of delivering software faster and with higher quality. Doing so successfully requires many different practices, behaviors, and technologies. 
 
-- Create two workflow files
+In this course, we focus on using workflows to deploy pull requests automatically to a staging environment, and to deploy all merged commits to production. We will create two workflows to deploy our app to an AWS S3 bucket automatically.
+
+
+## What you'll learn
+
+We'll answer common questions like:
+- What is Continuous Delivery?
+- What is a workflow file and how are workflows used with GitHub Actions?
+- What are deployment triggers and how are they used with deployments?
+- How can secrets store tokens?
+
+
+And when you're done you'll be able to:
+- Use GitHub Actions and create workflow files
 - Configure AWS S3 for deployment
```

Comment:
This course is not going to be limited to AWS S3.  We currently have a workflow that supports this exact course being sent to Azure as well.

Reply:
OK, i'll make that less specific